lang/cs/Org.Apache.REEF.Network/Group/Driver/Impl/GroupCommunicationMessage.cs

```diff
@@ -46,7 +36,7 @@ private GroupCommunicationMessage()
             string source,
             string destination,
             T message)
-            : base(groupName, operatorName, source, destination)
+            : base(groupName, operatorName, source, destination, typeof(T))
```

Comment:
I see. This is for the purpose of determining the type when the object is up-cast into a `GeneralGroupCommunicationMessage`, not for use when we get an object of type `GroupCommunicationMessage<T>`.